MatrixSDK/JSONModels/Login/MXLoginSSOFlow.m

```diff
@@ -16,7 +16,7 @@
 
 #import "MXLoginSSOFlow.h"
 
-NSString *const MXLoginSSOFlowIdentityProvidersKey = @"identity_providers";
+NSString *const MXLoginSSOFlowIdentityProvidersKey = @"org.matrix.msc2858.identity_providers";
```

Comment:
Maybe create another key: `MXLoginSSOFlowMsc2858IdentityProvidersKey`. In the future `MXLoginSSOFlowIdentityProvidersKey` will have to coexist with the value `"identity_providers"`.
But this change could be done later